loggly.go

```diff
@@ -92,7 +92,7 @@ func New(token string, tags ...string) *Client {
 // Send buffers `msg` for async sending.
 func (c *Client) Send(msg Message) error {
 	if _, exists := msg["timestamp"]; !exists {
-		msg["timestamp"] = time.Now().UnixNano() / int64(time.Millisecond)
+		msg["timestamp"] = time.Now().UTC().Format(time.RFC3339)
```

Comment:
`time.Time` is marshalled with RFC3339 by default. I would leave out formatting here and simply set the timestamp to a time.Time value. See http://golang.org/pkg/time/#Time.MarshalJSON

I'm not sure if it is required to drop the timezone information and only send UTC times. The timezone can be interesting information to have in the logs.